libs/features/mas/src/inline-price.js

```diff
@@ -255,15 +290,24 @@ export class InlinePrice extends HTMLSpanElement {
         const options = service.collectPriceOptions(overrides, this);
         if (!options.wcsOsi.length) return false;
 
-        /*
-        Commented out until issues in content with manually added tax labels are resolved
-
-        if (!this.masElement.dataset.displayTax) {
-            // set default value for displayTax if not set neither in OST nor in price URL
-            options.displayTax =
-                (await this.resolveDisplayTax(service, options)) || false;
+        const ffDefaultFlag = getMetadata(FF_DEFAULT_FLAG) === 'true';
```

Comment:
the metadata logic belongs to Milo, not to MAS. The feature flag should be resolved earlier when the commerce is enabled first by merch.js and set on `mas-commerce-service`  somehow.
We should avoid moving Milo concept in MAS.
fyi: @3ch023 @npeltier

Reply:
Actually first I tried it like that  andI was totally not happy with the solution. And now I see what I did wrong. That flag should be set in `mas-commerce-service`, I better don't say what I did. :D Will fix it.